lib/active_model/serializer/configuration.rb

```diff
@@ -7,9 +7,19 @@ module Configuration
       # Configuration options may also be set in
       # Serializers and Adapters
       included do |base|
-        base.config.array_serializer = ActiveModel::Serializer::ArraySerializer
-        base.config.adapter = :attributes
-        base.config.jsonapi_resource_type = :plural
+        config = base.config
+        config.collection_serializer = ActiveModel::Serializer::CollectionSerializer
+
+        def config.array_serializer=(collection_serializer)
```

Comment:
Wasn't sure how it would work in context so didn't bother.  What would the receiver be?

B mobile phone

> On Oct 7, 2015, at 5:12 AM, L. Preston Sego III <notifications@github.com> wrote:
> 
> In lib/active_model/serializer/configuration.rb:
> 
> > @@ -7,9 +7,19 @@ module Configuration
> >        # Configuration options may also be set in
> >        # Serializers and Adapters
> >        included do |base|
> > -        base.config.array_serializer = ActiveModel::Serializer::ArraySerializer
> > -        base.config.adapter = :attributes
> > -        base.config.jsonapi_resource_type = :plural
> > +        config = base.config
> > +        config.collection_serializer = ActiveModel::Serializer::CollectionSerializer
> > +
> > +        def config.array_serializer=(collection_serializer)
> why not use alias?
> 
> —
> Reply to this email directly or view it on GitHub.
>